app/views/RoomFollowView/index.js

```diff
@@ -0,0 +1,308 @@
+		} else {
+			this.actionSheetOptions.push(I18n.t('Mute'));
+		}
+		this.setState({ userLongPressed: user });
+		vibrate();
+		this.showActionSheet();
+	}
+
+	showActionSheet = () => {
+		ActionSheet.showActionSheetWithOptions({
+			options: this.actionSheetOptions,
+			cancelButtonIndex: this.CANCEL_INDEX,
+			title: I18n.t('Actions')
+		}, (actionIndex) => {
+			this.handleActionPress(actionIndex);
+		});
+	}
+
+	fetchFollowers = async() => {
+		const {
+			followers, isLoading, end, username
+		} = this.state;
+		const { user, navigation } = this.props;
+		if (isLoading || end) {
+			return;
+		}
+
+		this.setState({ isLoading: true });
+		try {
+			let followersResult = {};
+			if (navigation.getParam('follow') === 'Followers') {
+				followersResult = await RocketChat.getFollowers(username);
+			} else {
+				followersResult = await RocketChat.getFollowing(username);
+			}
+			/*
+			 Both followers and following are named as followers.
+			*/
+			let newFollowers = [];
+			const resultPromises = [];
+			let results = [];
+			const followersArray = Object.keys(followersResult);
+			for (let i = 0; i < followersArray.length; i += 1) {
+				resultPromises.push(RocketChat.getUserInfo(followersArray[i]));
+			}
+			await Promise.all(resultPromises).then((resultsTemp) => {
+				results =	resultsTemp.map(el => el.user);
+			});
+			newFollowers = results;
+
+			const followingOfTheUser = await RocketChat.getFollowing(user.username);
+			this.setState({
+				followers: followers.concat(newFollowers || []),
+				followingOfTheUser,
+				isLoading: false,
+				end: newFollowers.length < PAGE_SIZE
+			});
+		} catch (error) {
+			console.log('TCL: fetchFollowers -> error', error);
+			this.setState({ isLoading: false });
+		}
+	}
+
+	updateRoom = () => {
+		if (this.rooms.length > 0) {
+			const [room] = this.rooms;
+			this.setState({ room });
+		}
+	}
+
+	goRoom = async({ rid, name }) => {
+		const { navigation } = this.props;
+		await navigation.popToTop();
+		navigation.navigate('RoomView', { rid, name, t: 'd' });
+	}
+
+	handleMute = async() => {
+		const { rid, userLongPressed } = this.state;
+		try {
+			await RocketChat.toggleMuteUserInRoom(rid, userLongPressed.username, !userLongPressed.muted);
+			Toast(I18n.t('User_has_been_key', { key: userLongPressed.muted ? I18n.t('unmuted') : I18n.t('muted') }));
+		} catch (e) {
+			log('handleMute', e);
+		}
+	}
+
+	handleActionPress = (actionIndex) => {
+		switch (actionIndex) {
+			case this.MUTE_INDEX:
+				this.handleMute();
+				break;
+			default:
+				break;
+		}
+	}
+
+	renderSearchBar = () => (
+		<SearchBox onChangeText={text => this.onSearchChangeText(text)} testID='room-followers-view-search' />
+	)
+
+	renderSeparator = () => <View style={styles.separator} />;
+
+	renderItem = ({ item }) => {
+		const { baseUrl, user } = this.props;
+		const { followingOfTheUser } = this.state;
+		let follow = false;
+		if (followingOfTheUser[item._id] === '') {
```

Comment:
This can be re written as: var follow = (followingOfTheUser[item._id] === '') ? true : false

That seems to be the react native standard for logic like this